src/assets/scss/style.scss

```diff
@@ -15,3 +16,39 @@ body {
 .hover {
   color: $hovercolor;
 }
+
+* {
+  box-sizing: border-box;
+  outline: none;
+}
+
+.app-container {
+  width: 100%;
+  max-width: 81.25rem;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.show {
```

Comment:
Create a loader component that renders the framework's loader and overrides its styles.